OTAnalytics/plugin_datastore/track_store.py

```diff
@@ -571,18 +557,16 @@ def cut_with_section(
             for track_id, intersection_points in intersection_points.items()
         }
         tracks_to_cut = list(cut_indices.keys())
-        cut_tracks_df = self._dataset.loc[:, tracks_to_cut, :].copy()
+        cut_tracks_df = self._dataset.loc[tracks_to_cut].copy()
         index_as_df = cut_tracks_df.index.to_frame(
-            name=[LEVEL_CLASSIFICATION, LEVEL_TRACK_ID, LEVEL_OCCURRENCE]
-        )
-        index_as_df["cumcount"] = index_as_df.groupby(level=LEVEL_TRACK_ID).transform(
-            "cumcount"
+            name=[track.TRACK_ID, track.OCCURRENCE]
         )
+        index_as_df["cumcount"] = index_as_df.groupby(level=0).transform("cumcount")
```

Comment:
Is there a difference between `...groupby(level=0)` and `...groupby(level=LEVEL_TRACK_ID)`?

Reply:
No there is no difference, as long as the MultiIndex is named~